lighthouse-treemap/types/treemap.d.ts

```diff
@@ -0,0 +1,37 @@
+// import _Util2 = require('webtreemap-cdt'); // TODO: types.
+import _Util = require('../app/src/util.js');
+import {RootNodeContainer as _RootNodeContainer} from '../../lighthouse-core/audits/script-treemap-data';
+import {Node as _Node} from '../../lighthouse-core/audits/script-treemap-data';
```

Comment:
Maybe should move these types to here, and import in the audit?

I'm having trouble with imports/sharing here.